pom.xml

```diff
@@ -13,18 +13,33 @@
 					<version>3.8.1</version>
 					<executions>
 						<execution>
-							<id>java11</id>
-							<phase>compile</phase>
+							<id>java11-compile</id>
 							<goals>
 								<goal>compile</goal>
 							</goals>
+							<configuration>
+								<release>11</release>
+								<jdkToolchain>
+								<version>11</version>
+								</jdkToolchain>
+								<compileSourceRoots>
+								<compileSourceRoot>${project.basedir}/src/main/java11</compileSourceRoot>
+								</compileSourceRoots>
+								<multiReleaseOutput>true</multiReleaseOutput>
+							</configuration>
+						</execution>
+						<execution>
+							<id>java11-testcompile</id>
+							<goals>
+								<goal>testCompile</goal>
+							</goals>
 							<configuration>
 								<release>11</release>
 								<jdkToolchain>
 									<version>11</version>
 								</jdkToolchain>
 								<compileSourceRoots>
-									<compileSourceRoot>${project.basedir}/src/main/java11</compileSourceRoot>
+									<compileSourceRoot>${project.basedir}/src/test/java11</compileSourceRoot>
```

Comment:
should this also add ${project.basedir}/src/main/java11 so if you have an override java 11 file for a java 1.8 file the correct file will be tested